pkg/util/utils.go

```diff
@@ -180,23 +176,23 @@ func TeePrefix(prefix string, r io.Reader, w io.Writer, logger func(format strin
 
 // ReplaceChars returns a copy of the src slice with each string modified by the replacer
 func ReplaceChars(src []string, replacer *strings.Replacer) []string {
-	ret := make([]string, len(src))
+	returnSlice := make([]string, len(src))
```

Comment:
Just `ret` is fine: `returnSlice` is not necessarily more readable. 

For variable naming, omit words that are obvious given a variable's type declaration or otherwise clear from the surrounding context.

For go specifically, the length of the variable name should be proportional to it's scope: if the variable scope is only 1 or 2 lines, it should only be 1 or 2 characters. Global constants should have long names as they will require context whenever referenced.

Reply:
alright, fixed

Reply:
fixed.